config/locales/en.yml

```diff
@@ -57,30 +63,30 @@ en:
   address: 'Address'
   city: 'City'
   state: 'State'
+  zip_where_can_drive: 'Zip where you can drive'
```

Comment:
Good call. We'll have to find a way to support that, may require an addition to the User model. Mind if we add in a separate PR?